src/main/java/main/internet/shop/security/filter/AuthorisationFilter.java

```diff
@@ -21,28 +21,32 @@
     private static final String USER_ID = "userId";
     private static final Injector injector =
             Injector.getInstance("main.internet.shop");
-    private Map<String, List<UserRole>> protectedUrls;
+    private Map<String, List<UserRole.RoleName>> protectedUrls;
     private UserService userService = (UserService)
             injector.getInstance(UserService.class);
 
     @Override
     public void init(FilterConfig config) throws ServletException {
         protectedUrls = new HashMap<>();
-        protectedUrls.put("/user/all", List.of(UserRole.of("ADMIN")));
-        protectedUrls.put("/user/delete", List.of(UserRole.of("ADMIN")));
-        protectedUrls.put("/product/add", List.of(UserRole.of("ADMIN")));
-        protectedUrls.put("/admin/delete-order", List.of(UserRole.of("ADMIN")));
-        protectedUrls.put("/admin/products", List.of(UserRole.of("ADMIN")));
-        protectedUrls.put("/admin/product-delete", List.of(UserRole.of("ADMIN")));
-        protectedUrls.put("/product/all", List.of(UserRole.of("USER")));
-        protectedUrls.put("/product/buy", List.of(UserRole.of("USER")));
-        protectedUrls.put("/shopping-cart/products", List.of(UserRole.of("USER")));
-        protectedUrls.put("/product/success-buying", List.of(UserRole.of("USER")));
-        protectedUrls.put("/shopping-cart/delete-item", List.of(UserRole.of("USER")));
-        protectedUrls.put("/order/user-orders", List.of(UserRole.of("USER")));
-        protectedUrls.put("/order/create", List.of(UserRole.of("USER")));
-        protectedUrls.put("/order/details", List.of(UserRole.of("USER")));
-        protectedUrls.put("/main-menu", List.of(UserRole.of("USER")));
+        protectedUrls.put("/user/all", List.of(UserRole.RoleName.ADMIN));
+        protectedUrls.put("/user/delete", List.of(UserRole.RoleName.ADMIN));
+        protectedUrls.put("/product/add", List.of(UserRole.RoleName.ADMIN));
+        protectedUrls.put("/admin/delete-order", List.of(UserRole.RoleName.ADMIN));
+        protectedUrls.put("/admin/products", List.of(UserRole.RoleName.ADMIN));
+        protectedUrls.put("/admin/product-delete", List.of(UserRole.RoleName.ADMIN));
+        protectedUrls.put("/admin/orders", List.of(UserRole.RoleName.ADMIN));
+        protectedUrls.put("/admin/main-menu", List.of(UserRole.RoleName.ADMIN));
+        protectedUrls.put("/inject", List.of(UserRole.RoleName.ADMIN));
```

Comment:
in this controller we create our admin
seems like we can't go there at all now

let's make this controller available for anyone (remove it from here)